spec/support/fixtures/test/main.swift

```diff
@@ -1,3 +1,23 @@
-enum UnusedEnum {
-    case unused
+protocol RedundantProtocol {
 }
+
+class SomeClass: RedundantProtocol {
+    enum SomeEnum {
+        case usedCase
+        case unusedCase
+    }
+
+    var unusedProperty = 0
+    private var assignOnlyProperty = 0
+
+
+    public func methodWithRedundantPublicAccessibility(_ unusedParameter: Int) {
+        assignOnlyProperty = 0
+        _ = SomeEnum.usedCase
+    }
+
+    func unusedMethod() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/manicmaniac/danger-periphery/blob/eebb39add10edcb8769f633ddd123892cc2e1bfa/spec/support/fixtures/test/main.swift#L19"></span>Function 'unusedMethod()' is unused</td>
    </tr>
  </tbody>
</table>